### DISCUSSION TITLE
Which model is everyone using for embeddings?

### DISCUSSION THREAD

**Discussion by vdruts** [brianpetro/smart-connect#33] [Workflows]
Just curious, which models have you guys tested for notes/blocks and have you found any core difference in speed/recall/accuracy?

**Comment by miracletamer** [comment]
I have had vastly different results between OpenAI Text 3 Large and everything else - my chunking strategy is now so aligned with their current context window and response token limit that I am not even considering investing time in the others. It's a bit expensive but I don't want to risk using a smaller model API for fear of not maximizing my chances. I embed both my notes and blocks with this model and set a lower threshold for embedding of 10. My data is comprised of a lot of standards and specification so a lot of it is pre-chunked with a lot of small chunks that are relevant so warrant their own embedding in a document hierarchy - as I've curated the data I've learned a lot about how to deploy a successful chunking strategy

**Reply by vdruts** [reply]
Very interesting. I used to use Ada-2, but have a VERY large vault and it cost me a small fortune to re-embed, around $40 USD! I then switched to Small, and the cost is nothing now. Large is even more expensive than Ada-2 was! I can't imagine, given how often SC asks me to re-embed, especially during updates, how that would affect my wallet.

I do a lot of analysis for marketing, psychology related tasks and a better and more robust embedding would probably do wonders.

Most of my documents are very long and multi-layered.
